packages/client/src/views/GrantDetails.vue

```diff
@@ -1,5 +1,11 @@
 <template>
   <section>
+      <div class="grant-details-back-link" style="display: flex; justify-content: start" v-if="showBackLink">
+          <a @click="goBack" class="back-link">Back</a>
```

Comment:
Unless I'm missing it, this class is not used anywhere, so let's remove it.

Reply:
yes, that was use but not anymore, is romoved!